.cirrus.yml

```diff
@@ -35,6 +35,10 @@ task:
       test_host_script: |
         cd $ENGINE_PATH/src
         ./flutter/testing/run_tests.sh host_debug_unopt
+      test_web_engine: |
```

Comment:
this has to be postfixed `_script`, so `test_web_engine_script`.

Reply:
Thanks! Done.